change/@ni-nimble-components-99393253-07d1-4ec4-902b-41680bddd36e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@vivinkrishna-ni I see that there are a lot of changes made in the PR but I'm not sure if it is ready to re-review.

Similar to AzDo make sure to resolve comments if they have been addressed with a response if needed and reset reviewers when ready to review:

![image](https://github.com/ni/nimble/assets/1588923/5d1c7f40-d20a-40fd-8e6e-91c2ad38332b)

Just want to make sure you are aware of that workflow

Reply:
I was working on some comments, especially on [this one](https://github.com/ni/nimble/pull/1335#discussion_r1261476436). So I didn't re-requested for review.